<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filemanagerWidget</class>
 <widget class="QWidget" name="filemanagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="filemanagerTabWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>272</width>
     <height>477</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Directory</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="dirComboBox"/>
     </item>
     <item>
      <widget class="QTreeView" name="dirTreeView">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="fileListView">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="extComboBox">
       <item>
        <property name="text">
         <string>All Files (*.*)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text (*.txt)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jpeg (*.jpg)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Functions</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Property</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
